#!/usr/bin/python3
def raise_exception():
    raise TypeError("Wrong Type, try again")
